index.html

```diff
@@ -51,50 +51,57 @@
     <div class="container main-content" >
         <!-- row 1 -->
         <div id="content-search" class="row">
-            <div class="col-sm-4">
-                Text in row 1 column 1
+            <div id="row1-column1" class="col-sm-4">
+                <img id="card-image" src="">
+                
             </div>
-            <div class="col-sm-4" >
-    
+            <div id="row1-column2" class="col-sm-4" >
+              <div id="card-name-after-searched"> </div>
+              <div id="card-type-after-searched"> </div>
+              <div id="oracle-text"> </div>
+              <div id="power-and-toughness"> </div>
+
                 <!--Search bar-->
                     <form id="search-container" class="d-flex">
                         <input id="middle-search-text" type="text" class="form-control user-input" placeholder="Search">
                     <button id="submit-search" type="button" class="search-image"></button>
                     </form>  <!--End of search bar-->
-                     <div id="autocomplete-results"> 
+                    <div id="autocomplete-results" class=""> 
                         <ul id="autocomplete-results-list">
                              <a id="autocomplete-result-links"> </a>
 
                         </ul>
                     </div>
             </div>
-            <div class="col-sm-4">
-                Text in row 1 column 3
+            <div id="row1-column3" class="col-sm-4">
+                <div id="rarity"> </div>
+                <div id="artist"> </div>
+                
             </div>
         </div>
         <!-- row 2-->
         <div class="row">
-            <div class="col-sm-4">
-                Text in row 2 column 1
-                <img id="card-image" src="https://c1.scryfall.com/file/scryfall-cards/small/front/4/c/4cbc6901-6a4a-4d0a-83ea-7eefa3b35021.jpg?1625979257">
+            <div id="card-set" class="col-sm-4">   
             </div>
-            <div class="col-sm-4" >
-                Text in row 2 column 2
+            <a id="tcg-link" href="" target="_blank" class="d-none"> TCGplayer Link </a>
```

Comment:
target  _blank is the right choice for these links.  As we'd rather have them open new window/tabs and not redirect from our actual page.  So we can still have our page in context for the user.